migrator/src/main/java/com/google/phonenumbers/migrator/CommandLineMain.java

```diff
@@ -56,6 +59,24 @@
       description = "The BCP-47 country code the given phone number(s) belong to (e.g. 44)")
   String countryCode;
 
+  @ArgGroup()
+  OptionalParameterType optionalParameter;
+
+  static class OptionalParameterType {
+    @Option(names = {"-l", "--lenientExport"},
+        description = "boolean flag specifying that text files created after the migration process"
+            + " for standard recipe --file migrations should contain the migrated version of a given"
+            + " phone number when a migration has taken place on it, regardless of whether the"
```

Comment:
nit: maybe remove this part 'when a migration has taken place on it' to shorten it